snippets/csharp/GettingStarted.cs

```diff
@@ -76,4 +76,27 @@ public void FetchNodeInfo(BlockingBreezServices sdk)
         }
         // ANCHOR_END: fetch-balance
     }
+
+    // ANCHOR: logging
+    public void GettingStartedLogging()
```

Comment:
```suggestion
    public void GettingStartedLogging(BlockingBreezServices sdk)
```

Reply:
Fixed [ba0b87c](https://github.com/breez/breez-sdk-docs/pull/170/commits/ba0b87ccdf99fa2a1cd2373e6781a1e5d0ce9729), the `sdk = ...` was the issue

Reply:
Right!